encord/objects/classification_instance.py

```diff
@@ -95,6 +95,11 @@ def _last_frame(self) -> Union[int, float]:
         else:
             return self._parent.number_of_frames
 
+    @property
+    def range_list(self) -> List[List[int]]:
```

Comment:
perhaps smth like `frame_range_list` would be more clear? 
Also I wonder if using `List[List[int]]` is the best possible type here. Perhaps we could do List[Range] or at least List[Tuple[int, int]], to make the format a bit more obvious?

Reply:
Have changed the return type to List[Range].

Although I'm not sure if renaming the method to `frame_range_list` would be clearer, because its not necessarily a range of frames anymore. That said, alot of the code still assumes stuff are frames, so i'm okay either way.